docker/Dockerfile

```diff
@@ -17,27 +15,21 @@ RUN apt-get update && \
         zlib1g-dev \
         libncurses5-dev \
         libffi-dev \
-        libgdbm-dev && \
-   apt-get install -y default-mysql-server
-
-# Install nodejs and imagemagick
-WORKDIR /opt
-RUN wget https://nodejs.org/dist/v12.18.3/node-v12.18.3-linux-x64.tar.xz && \
-    tar xf node-v12.18.3-linux-x64.tar.xz && \
-    wget https://imagemagick.org/archive/binaries/magick && \
-    chmod +x magick && \
-    mv magick /usr/local/bin/magick
-
-ENV NODEJS_HOME=/opt/node-v12.18.3-linux-x64/bin
-ENV PATH=$NODEJS_HOME:$PATH
+        libgdbm-dev \
+        nodejs \
```

Comment:
NB!
---
If anyone decides to port this change, please, review that the Node.js version pulled is compatible or, better, yet, fix it to a specific version range (as we should do with all other dependencies, tbh).

Reply:
Outdated due to merge of #1430 - we no longer install imagemagick and Node.js directly (bullseye has imagemagick out-of-the-box, and Node.js is now copied over from a multi-step build).